Auth0/Handlers.swift

```diff
@@ -1,77 +0,0 @@
-import Foundation
```

Comment:
The contents of this file were moved to `AuthenticationHandlers.swift`. Since the internal `Response` type now makes use of Swift 6's typed throws, most handlers no longer need the last `catch`.